plugins/tinymcespellchecker.md

```diff
@@ -47,68 +49,186 @@ tinymce.init({
+This option specifies the URL of the server-side `ephox-spelling` service. For instructions on how to setup a Spell Checker Pro server-side component, see: the [server-side component installation guide]({{site.baseurl}}/enterprise/server/).
+
+> **Note:** `spellchecker_rpc_url` is **not** required when enabling this plugin via [{{site.cloudname}}]({{site.baseurl}}/cloud-deployment-guide/editor-and-features/)
+
+**Type:** `String`
+
+#### Example: `spellchecker_rpc_url`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_rpc_url: 'localhost/ephox-spelling'
+});
+```
 
 ### `spellchecker_languages`
-This optional setting allows specifying the languages that are available to the user, provided as a comma delimited string. The default value for this setting is:
 
+This option specifies the languages that are available to the user, provided as a comma delimited string.
+
+**Type:** comma-separated `String`
+
+**Default Value:**
 ````
-'US English=en,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt_BR,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
 ````
 
+#### Example: `spellchecker_languages`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_languages: 'US English=en_us,UK English=en_gb,Danish=da,Dutch=nl,Finnish=fi,French=fr,German=de,Italian=it,Norwegian=nb,Brazilian Portuguese=pt,Iberian Portuguese=pt_PT,Spanish=es,Swedish=sv'
+});
+```
+
 ### `spellchecker_language`
-This optional setting allows specifying the language code that will be used by default. This defaults to "en".
+
+This option specifies the default language used by Spell Checker Pro.
+
+**Type:** `String`
+
+**Default Value:** `'en_us'`
+
+#### Example: `spellchecker_language`
+
+```js
+tinymce.init({
+  selector: 'textarea',
+  plugins: 'tinymcespellchecker',
+  spellchecker_language: 'en_us'
+});
+```
 
 {% include configuration/languages.md %}
 
 ### `spellchecker_dialog`
-This optional setting allows specifying the mode of operation of the spellchecker. When set to true, the spellchecker will open a dialog that will check all of the document's contents at once. This also allows a user to step through each error, to ignore errors and accept recommended fixes. If the spellchecker dialog is enabled, the default as-you-type spellchecking will be disabled.
+
+This option enables or disables the spell checker dialog. When set to true, the [`spellcheckdialog` toolbar button](#toolbarbuttons) will open a dialog that will step the user through each potential spelling error in the document.
```

Comment:
This should be `spellchecker` toolbar button, as `spellcheckdialog` will always open the dialog.

Reply:
fixed